Gemfile

```diff
@@ -3,6 +3,8 @@ ruby '2.4.3'
 
 gem 'bootsnap', require: false
 gem 'devise'
+gem 'figaro'
```

Comment:
Warning, this gem is outdated. It fully works but it's a bit dangerous, we now use dotenv-rails: https://github.com/bkeepers/dotenv

Reply:
ok et mes secrets id facebook que j'ai mis sur heroku avce figaro sont quand meme protégés ?

Reply:
Yep, ça fait exactement la même chose